### PR TITLE
add profileShowDirectDeposit release toggle

### PR DESCRIPTION
**Pull request by rianfowler**
## Description of change
- adds direct deposit release toggle to flipper array 

## Testing planned

- will validate in staging today 

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: config/initializers/flipper.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@
       # default feautures to enabled for development and test only
       Flipper.enable(feature) if Rails.env.development? || Rails.env.test?
     end
-  rescue ActiveRecord::NoDatabaseError, ActiveRecord::StatementInvalid
```
Comment (annaswims) [inline config/initializers/flipper.rb]: catching the all of the db errors pre-migrations was a game of whack-a-mole

File: config/features.yml [hunk 2]
```diff
@@ -8,3 +8,6 @@ features:
     description: >
       On https://www.va.gov/find-locations/ enable veterans to search for Community care by showing that option in the "Search for" box.
       This toggle is owned by Rian
+  profile_show_direct_deposit:
```
Comment (rianfowler) [inline config/features.yml]: perfect thank you!

File: config/initializers/flipper.rb (hunk 1 above)
Comment (kfrz) [reply config/initializers/flipper.rb]: Maybe add a `Raven.tags_context(feature: 'flipper')` so we can easily find all flipper related errors in sentry?

Comment (annaswims) [reply config/initializers/flipper.rb]: All of the errors were when I tried to run a rake task like `rake db:migrate` or `rake db:setup` before the flipper before the flipper tables existed.  So, I don't expect any of the errors to be in prod.

File: config/features.yml (hunk 2 above)
Comment (annaswims) [reply config/features.yml]: One thing I'm wondering about now is if we want to have a singular toggle for "enable the API endpoint for direct deposit" and "show it in the UI".  If we had reason to shut it off we might want to make sure the backend also won't accept requests to update the banking info.

Comment (rianfowler) [reply config/features.yml]: I think separate toggles work for short lived toggles (release toggles) but and that long term toggles (ops toggle) would cover it both systems. 

We also have the option to poll the toggle values and live update ui.

**Comment by rianfowler** [comment]
I think this is ready for final approval.

**Pull request merged**